finance/proposal-calls/cycle3.md

```diff
@@ -0,0 +1,90 @@
+# Cycle III funding requests and ideas
+
+
+# What is different from the previous cycle?
+
+This cycle now includes funding from the NASA ROSES proposal, not just the Moore Grant. Process-wise, most of the below is similar to the Cycle II (at the time called “Phase II”), with some additional context and minor practical changes based on how Phase II progressed. The main difference in this cycle is the “hiring process” mentioned below. Concretely, in this cycle you can now propose a project that you do not necessarily wish to execute yourself, but that you would like Astropy to find someone to do the work. We hope this will serve both to get work done that our current community does not enjoy doing, as well as serving to expand our community. 
+
+
+# Funding goals
+
+Funding requests (FR) shall improve the Astropy Project. This includes code for the core repository, for coordinated or affiliated packages, infrastructure (e.g. testing, CI), tutorials, workshops, notebooks, community engagement, inclusion, diversity and empowerment, and anything else broadly Project related.
+
+FRs shall foster the Astropy Project goals of community, shared work, and cooperation and shall be proposed and executed in this manner.
+
+FRs need to clearly identify how the the funding will be used (e.g. "pay $YYY (US) to add Klingon Standard Time to astropy.time" or "organize astropy workshop to engage amateur astronomer community in Antarctica")
+
+
+# Funding amount
+
+We anticipate a maximum of **$400k** (US) to be available for this funding cycle. This is significantly larger than previous cycles. This means this cycle may be limited more by the available time of current community members rather than by the amount of funding available.  We therefore urge the community to **aim high**, either by ambitious and/or expensive proposals for their own work, or by bringing in new members via the “open hire” process that is new in this cycle (outlined below).
```

Comment:
```suggestion
We anticipate a maximum of **$400k** (US) to be available for this funding cycle. This is significantly larger than previous cycles. This means that approved funding requests may be limited more by the available time of current community members rather than by the amount of funding available.  We therefore urge the community to **aim high**, either by ambitious and/or expensive proposals for their own work, or by bringing in new members via the “open hire” process that is new in this cycle (outlined below).
```

Reply:
Alternative to the suggestion:
```suggestion
We anticipate a maximum of **$400k** (US) to be available for this funding cycle. This is significantly larger than previous cycles. This means that approved funding requests in this cycle may be limited more by the available time of current community members rather than by the amount of funding available.  We therefore urge the community to **aim high**, either by ambitious and/or expensive proposals for their own work, or by bringing in new members via the “open hire” process that is new in this cycle (outlined below).
```